testaddons/tests.go

```diff
@@ -581,3 +606,65 @@ func (options *TestAddonOptions) testTearDown() {
 		options.Logger.ShortInfo("No catalog to delete")
 	}
 }
+
+type Offering struct {
```

Comment:
Move this to the catalog types in cloud info for easier reuse. Needs a different name but we can come back to that before the merge. 
Might as list the optional inputs to and all the details about the inputs that are available, in case we need them later.